measured.gemspec

```diff
@@ -6,8 +6,8 @@ require 'measured/version'
 Gem::Specification.new do |spec|
   spec.name          = "measured"
   spec.version       = Measured::VERSION
-  spec.authors       = ["Kevin McPhillips"]
-  spec.email         = ["github@kevinmcphillips.ca"]
+  spec.authors       = ["Kevin McPhillips", "Jason Gedge", "Javier Honduvilla Coto"]
```

Comment:
🙇  Didn't do many changes, though! It's all your work, folks! But thanks for this! 💞

Reply:
This is me low-key asking you to keep helping and doing a great job.

![](https://i.imgur.com/SlWZwOZ.gif)